src/Text/Layout/Table/Spec/TableSpec.hs

```diff
@@ -9,58 +9,58 @@ import Text.Layout.Table.Spec.RowGroup
 import Text.Layout.Table.Style
 
 -- | Type used to specify tables.
-data TableSpec hSep vSep r c a
+data TableSpec colSep rowSep r c a
```

Comment:
I may be getting myself confused, but these seem backwards to me. I would expect colSep to be the separator between columns, i.e. vertical lines, and rowSep to be the separator between rows, i.e. horizontal lines.

If you check Test.hs, you can see that the second parameter for TableStyle is what determines the vertical lines.

Reply:
Thanks.  I'm not sure what I was thinking. :disappointed: